app/client/rest/channels.ts

```diff
@@ -278,7 +278,8 @@ const ClientChannels = (superclass: any) => class extends superclass {
     };
 
     viewMyChannel = async (channelId: string, prevChannelId?: string) => {
-        const data = {channel_id: channelId, prev_channel_id: prevChannelId};
+        // collapsed_threads_supported is not based on user preferences but to know if "CLIENT" supports CRT
+        const data = {channel_id: channelId, prev_channel_id: prevChannelId, collapsed_threads_supported: true};
```

Comment:
I don't understand the comment on line 281.  Should the value of `collapsed_threads_supported` be read from `config` ?  Why is `collapsed_threads_supported` defaulted to true?

Reply:
This is for letting the server know that client has code to support the CRT and handle the `channel viewed` functionality and It's not based on the config or user settings.

Reply:
Thanks @anurag6713.  I see that `collapsed_threads_supported` isn't documented in the API doc.  Perhaps, we can update the comments in code to your answer?